### ISSUE TITLE
Typo/mispelling in the .es Spanish version of the ethereum.org home page

### ISSUE THREAD

**Issue by A2be** [ethereum/ethereum-org-website#4877]
**Describe the bug**

The current word "implulsa" is a typo/misspelling. It is not a word in any language.

The correct word should be "impulsa", as it "powers the Ether cryptocurrency"

This is on line 6 of the page-index.json file.

[https://github.com/ethereum/ethereum-org-website/blob/c7546d01d825e6e4e8d037112d6f0327f947dc22/src/intl/es/page-index.json#L6](https://github.com/ethereum/ethereum-org-website/blob/c7546d01d825e6e4e8d037112d6f0327f947dc22/src/intl/es/page-index.json#L6)

**To Reproduce**

1. Go to '[...](https://ethereum.org/es/)'
2.  See error in the `Ethereum es la tecnología de gestión comunitaria que implulsa la criptomoneda ether (ETH) y miles de aplicaciones descentralizadas.`

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- Browser: [e.g. chrome, safari]
- Version: [e.g. 22]

**Smartphone (please complete the following information):**

- Browser: [e.g. stock browser, safari]
- Version: [e.g. 22]

**Additional context**

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by A2be** [comment]
I'd do the PR if I knew how, but am not sure I have the permissions or know-how to submit it.  Happy to learn if someone wants to show me how.

**Comment by vluna** [comment]
@A2be You can follow this [instructions](https://github.com/ethereum/ethereum-org-website#how-to-contribute) to set up your development environment and create the PR. If you have any questions I will be happy to help you. If you do not want to do it, I can create the PR for you 🙂

**Comment by A2be** [comment]
> @A2be You can follow this [instructions](https://github.com/ethereum/ethereum-org-website#how-to-contribute) to set up your development environment and create the PR. If you have any questions I will be happy to help you. If you do not want to do it, I can create the PR for you 🙂

Okay, cool.  Go ahead and assign the issue to me.  

Let's see if I can't get the switches and gears of github moving in such a way to make this simple fix and PR.

**Comment by A2be** [comment]
@vluna , I'm thru the initial steps, and up to this one:
**Set up your local environment (optional)**

1) If it is "optional", not sure I need all this next bit of complexity. Or do I?
Question:  is it "advisable" for me to do this set of steps? 

2) I use a WinOS computer.   I'm not sure how to `run` commands shown like:
`$ git remote add upstream https://github.com/ethereum/ethereum-org-website.git` on a WinOS machine.  
I do know how to use the command line window, and ran DOS stuff a lot back in the 1980s/90s, but those commands look Unix-ish to me, so I'm quite unsure if they'll run in Win, and how I need to set up the directories and permissions to get them to run smoothly on WinOS.

**Comment by vluna** [comment]
> @vluna , I'm thru the initial steps, and up to this one: **Set up your local environment (optional)**
> 
> 1. If it is "optional", not sure I need all this next bit of complexity. Or do I?
>    Question:  is it "advisable" for me to do this set of steps?
> 2. I use a WinOS computer.   I'm not sure how to `run` commands shown like:
>    `$ git remote add upstream https://github.com/ethereum/ethereum-org-website.git` on a WinOS machine.
>    I do know how to use the command line window, and ran DOS stuff a lot back in the 1980s/90s, but those commands look Unix-ish to me, so I'm quite unsure if they'll run in Win, and how I need to set up the directories and permissions to get them to run smoothly on WinOS.

1) I believe you can just skip that, I never set that up and I did not run into any problems. 
2) That is just a regular git command. I do not use windows, but you can use bash for it. Or maybe install windows linux subsystem?

**Comment by A2be** [comment]
Thanks @vluna .  Will skip that entire "optional" step.

But "bash" is not in my vocabulary, so hard to know what is involved, if I wanted to follow your next 
instruction:  "`you can use an external bash for it."

**Comment by vluna** [comment]
> Thanks @vluna . Will skip that entire "optional" step.
> 
> But "bash" is not in my vocabulary, so hard to know what is involved, if I wanted to follow your next instruction: " `you can use an external bash for it.` "
> 
> Also, although I've pulled up my fork `https://github.com/A2be/ethereum-org-website` on a browser tab, I'm not finding how to do the next step, in the section **Make awesome changes!**, which is `Create new branch for your changes`
> 
> Where does one do the "create new branch" step?

It means to create a new branch in git for your changes. The sample code is shown below in the step: `git checkout -b new_branch_name`

**Comment by vluna** [comment]
Hey @A2be! Were you able to figure it out?

**Comment by A2be** [comment]
I've been working at it.  Have done 
`winget` ...
and even got some sort of `git` for WinOS installed, 
so I now have Apps I can open called
`Git GUI`
`Git CMD` , and 
`Git Bash`  (although I still have no idea what "bash" is.

**Comment by A2be** [comment]
But I have not been able to figure out how to get all those various cmd line strings that are thrown around operating in my environment. 

I probably need to work in a coworking environment for a while, adjacent to an experienced Git and Github person, who is sorta kinda a little bit familiar with WinOS.    I need a sherpa.

Given I don't work in or around anyone like that, maybe it is time for me to just give up and let someone else fix that spelling error.  :(

**Comment by A2be** [comment]
So, bottom line, I still don't know how to `run` commands shown like:
`$ git remote add upstream https://github.com/ethereum/ethereum-org-website.git` on my WinOS machine.

**Comment by A2be** [comment]
So you should just go ahead and submit the fix.  Will make the ethereum.org website better in the quickest and most efficient way.

**Comment by A2be** [comment]
I got a LOT farther when I opened that `Git GUI` app on WindowsOS.

I might have even got that "git remote add upstream https://github.com/ethereum/ethereum-org-website.git` line to run, after I typed it in following a `$` cmd line prompt.

And I could even `browse` the file I want to update:  `src/intl/es/page-index.json`
where I want to fix the spelling of one word in line 6.

But I could never figure out how to edit just that single line.

And if I did, not sure what would then turn it into a PR on github.


But I ultimately could not figure out how to
